TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/ContentCache.php

```diff
@@ -237,11 +243,14 @@ public function getCachedSegment($uncachedCommandCallback, $typoScriptPath, $cac
     public function replaceCachePlaceholders(&$content, $addCacheSegmentMarkersToPlaceholders)
     {
         $cache = $this->cache;
+        $debugger = $this->debugger;
```

Comment:
das ist doch kein Javascript. Verwechselst du hier einfach was?? Einfach anstatt $debugger $this->debugger verwenden, that's all. Tut das nicht?